tests/BookReader/Toolbar/Toolbar.test.js

```diff
@@ -8,13 +8,14 @@ afterEach(() => {
 
 describe('createPopup', () => {
     test('calls window.open', () => {
-        const openSpy = sinon.spy(window, 'open');
+        const openSpy = sinon.stub(window, 'open');
```

Comment:
Let's rename these stub now (and anywhere they're used)

```suggestion
        const openStub = sinon.stub(window, 'open');
```